pubpub/pubpub
# PubEdge clickability/style updates

This is a quick one to refactor the behavior of the PubEdge/PubEdgeListingCard components so that the following behaviors:

- The whole card acting as a click target
- The greyscale-until-hover effect

Are only applied when the card has `inPubBody={true}`.

_Test plan:_
- Check that the Storybook fixtures behave as if in the Pub body
- Check that in the Connections dashboard the cards do not grey out, or act like giant click targets.
